File: sa.f
          real  SA
          integer h
          integer r
          real   pi
          pi = 3.1412
          do 20 h = 1,50,1
              do 30 r = 1,50,1
          
          SA = .5 * 2 * pi *r *h + .5 * 3.1412 * r**2
          print *,"Surface Area  = ",SA, "radius = ",r, "height = ",h
30         continue          
20         continue

          end

File: sa.cpp
#include <stdio.h>
#include <stdlib.h>
#include <cmath>
int main(){
float SA;
int r;
int h;
for (r=1;r<=50;r++){
for (h=1;h<=50;h++){
  SA = .5 * 2 * M_PI *r *h + .5 * M_PI * r*(exp(2));
  printf("\n%s%f%s%d%s%d","Surface Area ",SA," radius= ",r," height = ",h);

  }
  }
  }
